monitoring/rp2json.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
```

Comment:
Assuming this requires python 2? There are no plans to introduce python 3 anytime soon. Just wanted to make sure and ask.